include/k4a/k4atypes.h

```diff
@@ -681,8 +681,6 @@ typedef enum
 typedef enum
 {
     K4A_CALIBRATION_LENS_DISTORTION_MODEL_UNKNOWN = 0,   /**< Calibration model is unknown */
-    K4A_CALIBRATION_LENS_DISTORTION_MODEL_THETA,         /**< Calibration model is Theta (arctan) */
-    K4A_CALIBRATION_LENS_DISTORTION_MODEL_POLYNOMIAL_3K, /**< Calibration model Polynomial 3K */
```

Comment:
@rajeev-msft, what do you think about removing these values? We can always add them back if necessary.

@rabbitdaxi this is released so we need to mark them deprecated for 1 release before we can delete them.

Reply:
@wes-b  let us chat more about this